includes/modules/meta_tags.php

```diff
@@ -191,6 +191,9 @@
   $review_on = META_TAGS_REVIEW;
   //  case 'product_info':
   case (strstr($_GET['main_page'], 'product_') or strstr($_GET['main_page'], 'document_')):
+  if (empty($_GET['products_id'])) {
+    $_GET['products_id'] = 0;
+  }
```

Comment:
This check/set is done pretty late in the file. Is there any value in doing it earlier?

Reply:
(I'm not bothered if we don't need to do it earlier. Just checking what your thoughts are.)

Otherwise, I'm generally satisfied that we could squash these commits and then merge.

Reply:
The initial placement was based on where it was known it would be needed (viewing a product's `_info` page).  Having run the "test" that gave all of the notices, I would think that other similar tests should be run with other conditions.  

Unfortunately, making that an exhaustive test seems a bit (unless there is some utility that can be used to simulate all of the potential combinations one at a time. Something I guess I could put together as some code to be fed all of the parameters and some level of conditions to each parameter having both good and bad conditions that would access the index.php repetitively until completion). 

Further, it seemed that if one of the other `case` statements expected to specifically handle `$_GET['products_id']`, that I would have come across some issue at some point and/or the case would have something within it.  The issue there instead being to test what problems occur when it is provided with values in the database and then when not in the database.

Basically, I expect that I will tinker with parameters on the URL regarding known/expected parameters and associated values, though doubt that I will get as creative as has been seen in the wild. At the moment I can't say that the statement would hurt being earlier in processing/evaluation (ie. init_sanitize) or be acceptable to other code.

I can squash the commits and force push if so desired.